src/main/java/com/oltpbenchmark/benchmarks/twitter/TwitterLoader.java

```diff
@@ -287,7 +287,7 @@ protected void loadFollowData(Connection conn, int lo, int hi) throws SQLExcepti
 
                         total++;
                         batchSize++;
-                        f++;
+
```

Comment:
Is this the desired behavior? I don't know; probably @apavlo would know. Now `f` can increase without a new followee.

Reply:
ah so here is the deal... when i reconfigured the abstract base test case to apply a smaller default scale factor to all tests (eg `.01`) i noticed some odd behavior in the twitter test.  my comment at the time said this... `fixing bug in twitter loader that only showed up with very low scale factors causing infinite loop`.  you can see that prior to this change it would have been possible for `f` to never have been incremented thus causing the infinite loop.